config/locales/ypk.yml

```diff
@@ -0,0 +1,135 @@
+﻿ypk:
+
+# Default page title of search results page
+  default_serp_title: "%{query} - %{site_name} Ivaghillghem pillghi"
+
+# Labels for the search box
+  search: "Ivaghi"
+  search_label: "Kanaghtigu akuzita ivaghyukan"
+
+# Labels for the navigation bar
+  everything: "Qamalluku"
+  images: "Igaqellghet"
+  videos: "Muvingllat"
+  show_more: "Wata"
+
+# Labels to change the sort order by date or relevance
+  by_date: "Qavnganghhaghtat"
+  by_relevance: "Itagnallgha naalkegkam"
+
+# Labels to narrow results by a preset timeframe or a custom date range, or clear filters ('Before' is the label for custom date range with only a 'To' date set.)
+  all_time: "Alngunak qakun"
```

Comment:
Please add at least a placeholder for `  refine_your_search:` as a new first item in this section. Localized text is not required, but would be nice. Having the placeholder will allow the test to pass.

example at https://github.com/GSA/search-gov/blob/main/config/locales/ko.yml#L23